pkg/apis/serving/v1alpha1/route_types.go

```diff
@@ -179,6 +179,9 @@ type RouteStatus struct {
 	// even if the controller failed to process the spec and create the Revision.
 	// +optional
 	ObservedGeneration int64 `json:"observedGeneration,omitempty"`
+
+	// Ready holds the value of the most recent RouteContitionReady or false.
+	Ready bool `json:"ready"`
```

Comment:
I don't want to introduce this to our API just for `kubectl`.  Conditions are a pretty fundamental aspect of Kubernetes, which IMO kubectl's custom column support should support.  I'll ping some K8s folks.

Reply:
Fair enough - I'll also do some digging to figure out where k8s is at WRT to supporting this.

Reply:
Updated with a fix - we can write a jsonpath template

Reply:
This is neat, TIL.  I think we want this for all of our objects, include Build (but w/Succeeded)